generators/server/files.js

```diff
@@ -1,7 +1,15 @@
+const path = require('path');
+const { logger } = require('generator-jhipster/cli/utils');
+const fse = require('fs-extra');
+
 module.exports = {
     writeFiles
 };
 
 function writeFiles() {
-    this.copy('_dummy-server.txt', 'dummy-server.txt');
+    // TODO: Implement basic code for file generation.
+    fse.ensureLink(
```

Comment:
why do you use fs-extra if jhipster already gives you Api to manage files?

Reply:
I did not bother using the API because this is currently temporary: the code should be removed when someone makes a PR with the server generation (using .ejs as well I suppose). By the time we get basic templating for the server, we'll be using jhipster's api

Reply:
Ok, then keep hold my code

Reply:
Sorry, I did not quite understand what you meant there

Reply:
In my code, "this.copy" jhipster api does the same thing of your code. So keep it as template, because it is the standard for adding files that can be used in future for new PR features. Is it all clear?